torch/linalg/__init__.py

```diff
@@ -140,6 +140,111 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+svd = _add_docstr(_linalg.linalg_svd, r"""
+linalg.svd(input, full_matrices=True, compute_uv=True, out=None) -> (Tensor, Tensor, Tensor)
+
+This function returns a namedtuple ``(U, S, Vh)`` which is the singular value
+decomposition of a input real matrix or batches of real matrices :attr:`input` such that
+:math:`input = U \times diag(S) \times V^H` (where :math:`V^H` is ``Vh``).
+
+.. warning:: **Differences with** :meth:`~torch.svd`:
+
+             * :attr:`full_matrices` is the opposite of
+               :meth:`~torch.svd`'s :attr:`some`. Note that default value
+               for both is ``True``, so the default behavior is effectively
+               the opposite.
+
+             * it returns ``Vh``, whereas :meth:`~torch.svd` returns
+               ``V``. The result is that when using :meth:`~torch.svd` you
+               need to manually transpose and conjugate ``V`` in order to
+               reconstruct the original matrix.
+
+             * If :attr:`compute_uv=False`, it returns empty tensors (i.e.,
+               with 0 elements) for ``U`` and ``V``, whereas
+               :meth:`~torch.svd` returns zero-filled tensors.
+
+             **Differences with** ``numpy.linalg.svd``:
+
+             * if :attr:`compute_uv=False` it returns ``(empty_tensor, S, empty_tensor)``,
+               whereas numpy returns ``S``.
+
+
+The dtype of ``U`` and ``V`` is the same as the ``input`` matrix. The dtype of
+``S`` is always real numbers, even if ``input`` is complex.
+
+If :attr:`full_matrices` is ``False``, the method returns the reduced singular value decomposition
+i.e., if the last two dimensions of :attr:`input` are ``m`` and ``n``, then the returned
+`U` and `V` matrices will contain only :math:`min(n, m)` orthonormal columns.
+
+If :attr:`compute_uv` is ``False``, the returned `U` and `V` will be None.:attr:`full_matrices` will
```

Comment:
This sentence needs to be updated since they will now be empty tensors with no elements with the same device and dtype as :attr:\`input\`. 

The second part of this sentence could be: 

"When :attr:\`compute_uv\` is False :attr:\`full_matrices\` has no effect."